pages/platform/ai/app_tuto_01_gradio_sketch_recognition/guide.en-asia.md

```diff
@@ -0,0 +1,260 @@
+---
+title: AI Apps - Tutorial - Deploy a Gradio app for sketch recognition
+slug: apps/tuto-gradio-sketch-recognition
+excerpt: How to build and use a custom Docker image containing a Gradio application
+section: AI Apps tutorials
+order: 01
+---
+
+**Last updated 23th June, 2022.**
+
+## Objective
+
+The purpose of this tutorial is to deploy an application for sketch recognition using a trained model.
+
+The use case is **handwritten digits recognition**, based on the [MNIST dataset](http://yann.lecun.com/exdb/mnist/).
+
+In order to do this, you will use [Gradio](https://gradio.app/), an open-source Python library which is a quick way to expose and use Machine Learning models. You will also learn how to build and use a custom Docker image for a Gradio application.
+
+Overview of the app:
+
+![Overview](images/overview-gradio-app.png){.thumbnail}
+
+## Requirements
+
+- Access to the [OVHcloud Control Panel](https://ca.ovh.com/auth/?action=gotomanager&from=https://www.ovh.com/asia/&ovhSubsidiary=asia).
+- An AI Apps project created inside a [Public Cloud project](https://www.ovhcloud.com/asia/public-cloud/) in your OVHcloud account.
+- A [user for AI Apps](https://docs.ovh.com/asia/en/publiccloud/ai/users/).
+- [Docker](https://www.docker.com/get-started) installed on your local computer.
+- Some knowledge about building image and [Dockerfile](https://docs.docker.com/engine/reference/builder/).
+- You also should have followed the steps of the *Image Classification with MNIST dataset* notebook from the [GitHub repository](https://github.com/ovh/ai-training-examples/blob/main/notebooks/computer-vision/image-classification/tensorflow/weights-and-biases/notebook_Weights_and_Biases_MNIST.ipynb). You will be able to train and save your model.
```

Comment:
@eleapttn je pense qu'il faut rajouter le lien vers le tuto pour expliquer comment créer le notebook et le lancer : https://docs.ovh.com/gb/en/publiccloud/ai/notebooks/tuto-weights-and-biases/